### PR TITLE
Add initial content for the Build Your First Integration guide

### PR DESCRIPTION
**Pull request by Nashaath**
## Purpose
Add initial content for the `Build Your First Integration` guide.

### CODE REVIEW

File: en/docs/integrations/build-your-first-integration.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+5. In the **Configurations** panel, fill in the details as follows to configure the integration:
+    1. Click **Connect to GitHub**.
+    2. In the authorization request page that opens, click **Authorize wso2** to allow WSO2 to access your GitHub account. 
+    3. In the GitHub access confirmation page that opens, enter your GitHub password and click **Confirm password**.
+    4. In the **GitHub Repository URL** field, select the repository for which you want to track new issues.
+    5. Click **Connect to Google Sheets**, and then click **Proceed**.'
+    6. Select the Google account you want to use in the integration.
+    7. In the access request page that opens, click **Allow** to proceed with the selected Google account.
+    8. In the **Spreadsheet Name** field, select `Sample GitHub Integration`.
+    9 . In the **Worksheet Name** field, enter `Sheet1`.
```
Comment (anupama-pathirage) [inline en/docs/integrations/build-your-first-integration.md:28]: It seems the number `9` is not properly added.

File: en/docs/integrations/build-your-first-integration.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# Build Your First Integration
+Choreo allows you to efficiently build integrations by connecting APIs, events, and streams. Choreo also provides a comprehensive set of prebuilt integration templates that you can select and use depending on your requirement.
+
+This quick start guide walks you through the steps to quickly configure and try out the **GitHub New Issue to Google Sheets Row** integration in Choreo.
+
+## Prerequisites:
+- A GitHub account to connect to in the integration configuration.
+- A Google account to connect to in the integration configuration.
+- A blank spreadsheet named `Sample GitHub Integration` created in the Google account. This is the spreadsheet name to use in the integration configuration.
```
Comment (anupama-pathirage) [inline en/docs/integrations/build-your-first-integration.md:9]: Should we remove the verb `created` here. Seems the sentence is dramatically incorrect.

File: en/docs/integrations/build-your-first-integration.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+Now the **GitHub New Issue to Google Sheets Row** integration is successfully configured and running.
+
+## Step 2: Try out the integration 
+Follow the procedure below to try out the integration:
+
+1. Go to the GitHub repository you specified when configuring the integration and create two new issues.
+2. Go to the Choreo Console and take a look at the **Execution History** of the configured integration. 
+   ![Execution History](/assets/img/integrations/execution-history.png)
+   You will see two executions. These executions are a result of creating new issues in the GitHub repository.
```
Comment (RukshiW) [inline en/docs/integrations/build-your-first-integration.md:52]: Shall we avoid future tense and say "You can see two executions."?

File: en/docs/integrations/build-your-first-integration.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+3. Go to the `Sample GitHub Integration` spreadsheet in your Google Drive and take a look at `Sheet1`. 
+   ![](/assets/img/integrations/spreadsheet.png)
+
+    You will see two entries with details of the GitHub issues you created.
```
Comment (RukshiW) [inline en/docs/integrations/build-your-first-integration.md:57]: Shall we avoid future tense and say "You can see two entries".

File: en/docs/integrations/build-your-first-integration.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
+# Build Your First Integration
+Choreo allows you to efficiently build integrations by connecting APIs, events, and streams. Choreo also provides a comprehensive set of prebuilt integration templates that you can select and use depending on your requirement.
+
+This quick start guide walks you through the steps to quickly configure and try out the **GitHub New Issue to Google Sheets Row** integration in Choreo.
+
+## Prerequisites:
```
Comment (anupama-pathirage) [inline en/docs/integrations/build-your-first-integration.md:6]: The repetitive words `integration configuration` is a bit odd and confusing to me. Can't we avoid that repetition and simply say:
```
Following are the prerequisites for the integration configuration.
- A GitHub account.
- A Google account.
- A blank spreadsheet named Sample GitHub Integration with a default worksheet named Sheet1 created in the above Google account.  That spreadsheet name and worksheet name will be used in the configuration.
```

File: en/docs/integrations/build-your-first-integration.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+- A GitHub account to connect to in the integration configuration.
+- A Google account to connect to in the integration configuration.
+- A blank spreadsheet named `Sample GitHub Integration` created in the Google account. This is the spreadsheet name to use in the integration configuration. 
+  The spreadsheet will have a default worksheet named `Sheet1`. This is the worksheet name to use in the integration configuration.
```
Comment (RukshiW) [inline en/docs/integrations/build-your-first-integration.md:10]: Shall we avoid future tense and replace "will have" with "includes"?

File: en/docs/integrations/build-your-first-integration.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+1. Access the Choreo Console via [https://console.choreo.dev/](https://console.choreo.dev/).
+2. Sign in using either your GitHub or Google credentials.
+3. Go to the **Integrations** card and click **Get Started**. Now you are on the **Integrations** page, where you can start with a prebuilt integration or create your own.
+4. Go to the **GitHub New Issue to Google Sheets Row** card and click **Use This**.
+5. In the **Configurations** panel, fill in the details as follows to configure the integration:
+    1. Click **Connect to GitHub**.
+    2. In the authorization request page that opens, click **Authorize wso2** to allow WSO2 to access your GitHub account. 
+    3. In the GitHub access confirmation page that opens, enter your GitHub password and click **Confirm password**.
+    4. In the **GitHub Repository URL** field, select the repository for which you want to track new issues.
+    5. Click **Connect to Google Sheets**, and then click **Proceed**.'
```
Comment (anupama-pathirage) [inline en/docs/integrations/build-your-first-integration.md:24]: Extra back tick at the end?

File: en/docs/integrations/build-your-first-integration.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+## Step 2: Try out the integration 
+Follow the procedure below to try out the integration:
+
+1. Go to the GitHub repository you specified when configuring the integration and create two new issues.
+2. Go to the Choreo Console and take a look at the **Execution History** of the configured integration. 
+   ![Execution History](/assets/img/integrations/execution-history.png)
+   You will see two executions. These executions are a result of creating new issues in the GitHub repository.
+
+3. Go to the `Sample GitHub Integration` spreadsheet in your Google Drive and take a look at `Sheet1`.
```
Comment (anupama-pathirage) [inline en/docs/integrations/build-your-first-integration.md:54]: When referring to this spreadsheet, some places we say `spreadsheet in your Google Drive` and in some other places we say `spreadsheet in Google Account` . Use one convention.

File: en/docs/integrations/build-your-first-integration.md [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+2. Go to the Choreo Console and take a look at the **Execution History** of the configured integration. 
+   ![Execution History](/assets/img/integrations/execution-history.png)
+   You will see two executions. These executions are a result of creating new issues in the GitHub repository.
+
+3. Go to the `Sample GitHub Integration` spreadsheet in your Google Drive and take a look at `Sheet1`. 
+   ![](/assets/img/integrations/spreadsheet.png)
+
+    You will see two entries with details of the GitHub issues you created.
+
+Congratulations! Now you have successfully configured and tried out the **GitHub New Issue to Google Sheets Row** integration.
```
Comment (anupama-pathirage) [inline en/docs/integrations/build-your-first-integration.md:59]: @Nashaath and @RukshiW Shall we use one similar convention for the beginning and the ending of each of the simipar pages? https://github.com/wso2/docs-choreo-dev/blob/dev/en/docs/services/create-your-first-service.md Page is following a different wordings and patterns.

File: en/docs/integrations/build-your-first-integration.md (hunk 1 above)
Comment (Nashaath) [reply en/docs/integrations/build-your-first-integration.md]: Fixed.

File: en/docs/integrations/build-your-first-integration.md (hunk 2 above)
Comment (Nashaath) [reply en/docs/integrations/build-your-first-integration.md]: Updated the prerequisites section.

File: en/docs/integrations/build-your-first-integration.md (hunk 5 above)
Comment (Nashaath) [reply en/docs/integrations/build-your-first-integration.md]: Updated the prerequisites section.

File: en/docs/integrations/build-your-first-integration.md (hunk 3 above)
Comment (Nashaath) [reply en/docs/integrations/build-your-first-integration.md]: Changed.

File: en/docs/integrations/build-your-first-integration.md (hunk 4 above)
Comment (Nashaath) [reply en/docs/integrations/build-your-first-integration.md]: Changed.

File: en/docs/integrations/build-your-first-integration.md (hunk 6 above)
Comment (Nashaath) [reply en/docs/integrations/build-your-first-integration.md]: Updated the prerequisites section.

File: en/docs/integrations/build-your-first-integration.md (hunk 7 above)
Comment (Nashaath) [reply en/docs/integrations/build-your-first-integration.md]: Fixed.

**Review by anupama-pathirage — APPROVED** [review]

File: en/docs/integrations/build-your-first-integration.md (hunk 8 above)
Comment (Nashaath) [reply en/docs/integrations/build-your-first-integration.md]: Used `Google account`.

File: en/docs/integrations/build-your-first-integration.md (hunk 9 above)
Comment (Nashaath) [reply en/docs/integrations/build-your-first-integration.md:58]: +1

**Pull request merged**